src/PrestaShopBundle/Translation/PrestaShopTranslatorTrait.php

```diff
@@ -53,27 +53,39 @@ public function trans($id, array $parameters = [], $domain = null, $locale = nul
             $legacy = $parameters['legacy'];
             unset($parameters['legacy']);
         }
+        $emptyParams = empty($parameters);
 
         if (empty($locale)) {
             $locale = null;
         }
 
-        $translated = parent::trans($id, [], $this->normalizeDomain($domain), $locale);
-
-        // @todo to remove after the legacy translation system has ben phased out
-        if ($this->shouldFallbackToLegacyModuleTranslation($id, $domain, $translated)) {
-            return $this->translateUsingLegacySystem($id, $parameters, $domain, $locale);
```

Comment:
The legacy fallback should be kept

Reply:
I think I removed it by mistake :/ Will re-add it thanks for the catch